charts/sn-platform/templates/openshift/scc-role.yaml

```diff
@@ -31,5 +31,5 @@ rules:
     resources:
       - securitycontextconstraints
     resourceNames:
-      - {{ include "pulsar.openshift.scc.name" . }}
+      - {{ default (include "pulsar.openshift.scc.name" . ) .Values.openshift.scc.name }}
```

Comment:
Can you wrap this into the template "pulsar.openshift.scc.name"? So the name of the SCC to create could also be controlled.

Reply:
Updated.